core/templates/domain/collection/read-only-collection-backend-api.service.ts

```diff
@@ -37,42 +59,45 @@ import { UrlInterpolationService } from
 export class ReadOnlyCollectionBackendApiService {
   constructor(
     private http: HttpClient,
+    private collectionObjectFactory: CollectionObjectFactory,
     private urlInterpolationService: UrlInterpolationService) {}
-  private _collectionCache = [];
-  private _collectionDetailsCache = [];
+  private _collectionCache: CollectionCache = {};
+  private _collectionDetailsCache: CollectionDetailsCache = {};
```

Comment:
Remove `_` here?

Reply:
Actually @seanlip was in favour of keeping the `_` along with private.